src/components/views/create/CreateProject.tsx

```diff
@@ -75,7 +74,7 @@ export type TInputs = {
 	[EInputs.impactLocation]?: string;
 	[EInputs.image]?: string;
 	[EInputs.draft]?: boolean;
-	[EInputs.addresses]: { [key: number]: string };
+	[EInputs.addresses]: IWalletAddress[];
```

Comment:
```suggestion
	[EInputs.addresses]: Record<string | number, IWalletAddress>;
```

Reply:
If I apply this change we will face some different type errors.